include/Megamix/Region.hpp

```diff
@@ -46,6 +46,7 @@ namespace Region {
     u32 RodataEnd();
     u32 DataEnd();
     u32 BssEnd();
+    u32 TickflowCommandSwitch();
```

Comment:
please separate this from the section locations since they're semantically different